dask_glm/utils.py

```diff
@@ -137,18 +138,32 @@ def sum(A):
     return A.sum()
 
 
+def is_dask_array_sparse(X):
+    """
+    Check using _meta if a dask array contains sparse arrays
+    """
+    return isinstance(X._meta, sparse.COO) or isinstance(X._meta, sparse.DOK)
```

Comment:
Following the discussion on deprecation of `DOK`. Should I test only if the matrix is `COO` ? 
I would like to support only `COO` instead of sparse matrix that is too vague. Wdyt @TomAugspurger  ?

Reply:
Try `sparse.SparseArray`. That's best for forward compat.